facades/PC/build.gradle

```diff
@@ -81,11 +81,33 @@ task game(type:JavaExec) {
     classpath project(':engine').configurations.runtime
 }
 
+task debug(type:JavaExec) {
```

Comment:
I don't know enough about gradle to be making those kinds of modifications, as I normally use maven.  The port number can be anything, and a decent IDE should pick it up.  I've changed it numerous times and my IDE picks up the change every time, I dunno about other IDEs, tho.  If anything, debuggers themselves are configurable.

I don't know if it's possible, but I'd like the debug task to take arguments and then go on to the next task, so there could be something like this:
`./gradlew debug port=1044,suspend=y game`
or this:
`./gradlew debug port=1033,suspend=y server`